curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64fad9cd2eeb1e7ca2ca8c8b.md

```diff
@@ -0,0 +1,357 @@
+---
+id: 64fad9cd2eeb1e7ca2ca8c8b
+title: Step 31
+challengeType: 0
+dashedName: step-31
+---
+
+# --description--
+
+Use arrow syntax to create an `addTaskToTaskContainer` function. Then move the `taskData.forEach()` and its content into it.
```

Comment:
When the project landed in the curriculum expansion repo, there was a suggestion about variable and function names not being descriptive enough. 

We chose to go with `addTaskToTaskContainer` because it describes the exact thing we want to do.

Reply:
I see, I'd argue `addTaskToTaskContainer` isn't as precise, especially when there's more than one task. Another idea that just now came to my mind is `updateTaskContainer`.

Either way feel free to resolve the conversation.

Reply:
Thanks @scissorsneedfoodtoo and @gikf 

I've changed them all to updateTaskContainer.